pytest-backend/pyproject.toml

```diff
@@ -0,0 +1,67 @@
+[tool.poetry]
+name = "pytest-exasol-backend"
+version = "0.1.0"
+description = ""
+authors = ["Mikhail Beck <mikhail.beck@exasol.com>"]
+readme = "README.md"
+packages = [{include = "exasol"}]
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4"
+pytest = ">=7,<9"
+exasol-saas-api = ">=0.6.0,<1.0.0"
+pytest-exasol-itde = { path = "../pytest-itde" }
+pytest-exasol-saas = { path = "../pytest-saas" }
```

Comment:
Are these references to pypi packages?

Reply:
That will complicate the process of making changes to the plugins. One would need to change the `pytest-itde` or `pytest-saas`, then make a release, then update the `pytest-backend` and make another release. So the `pytest_backend` will never use the latest versions of the other two plugins. Is there a particular issue with referencing other projects in the same repo as it is now?

Reply:
I am thinking of users using the plugins.
The users will use a pypi reference to `backend`, I assume. 
pypi will not be able to resolve the transitive references with `path = "..."`.

Reply:
Will it not? I will do some experiments.

Reply:
I built a wheel for the `pytest-exasol-backend` and installed it in a new venv. All 3 plugins are there, so it seems to work.